app/utils/prepareAnswer.js

```diff
@@ -281,8 +281,8 @@ function dailyStatsGraph(body, n) {
   dateLabel.reverse();
 
   let link = `https://quickchart.io/chart?c={type:'line',data:{labels:[${dateLabel}],`;
-  link += `datasets:[{label:'Daily new cases',data:[${data}],backgroundColor:'white', fill:false, borderColor:'red'}]}}`;
-
+  link += `datasets:[{label:'Daily new cases',data:[${data}],backgroundColor:'white', fill:false, borderColor:'red'}]},`;
+  link += `options: {title:{display: true,text: 'Daily confirmed cases graphical analysis for past ${minN} days'},}}`;
```

Comment:
You don't have to use the word `graphical` because it's obvious.

Reply:
Okay will remove it 👍

Reply:
thanks